data_generators/xcspec_extractor.py

```diff
@@ -5,7 +5,7 @@
 and prints out a bzl file that contains a single constant, `SETTINGS`,
 that can be used to access the xcspecs from starlark.
 
-Usage: xcspec_extractor.py /Application/Xcode.app/Contents/Developer 11.2.1
+Usage: xcspec_extractor.py /Application/Xcode.app/Contents/Developer 11.2.1 ../data/xcspecs.bzl  ../data/xcspec_evals.bzl
```

Comment:
Intentional?

Reply:
Yea, those are actual required parameters when running it directly from python. Probably could also document using `bazel run //data_generators:extract_xcspecs` instead